app/src/main/java/at/roteskreuz/stopcorona/screens/reporting/ReportingActivity.kt

```diff
@@ -115,7 +117,10 @@ fun Fragment.startReportingActivity(
     dateWithMissingExposureKeys: ZonedDateTime? = null
 ) {
     startFragmentActivity<ReportingActivity>(
-        fragmentName = ReportingPersonalDataFragment::class.java.name,
+        // using this empty fragment to make sure the fragment setting is done properly in code.
+        fragmentName = EmptyFragment::class.java.name,
```

Comment:
As discussed on slack, this approach is not robust, you rely on some hardcoded initialisation which is not good and it's more like a hack than the intended solution.

Please reimplement it in one of the following ways:
1.  Observe parameters from `ReportingRepository` in fragments instead of using getters.
2. Pass the parameter to arguments of the fragment if you need it during init.